.env.example

```
@@ -0,0 +1,18 @@
+POSTGRES_DB=porsesh
+
+# Optional: expose Postgres on a different host port (default 5432)
+# POSTGRES_PORT=5432
+
+# Optional: expose the app on a different host port (default 3000)
+# APP_PORT=3000
+
+# ── Clerk authentication ───────────────────────────────────────────────────
+# Get these from https://dashboard.clerk.com → your application → API Keys
+NEXT_PUBLIC_CLERK_PUBLISHABLE_KEY=pk_test_...
+CLERK_SECRET_KEY=sk_test_...
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The CLERK_SECRET_KEY key should go before the NEXT_PUBLIC_CLERK_PUBLISHABLE_KEY key

(UnorderedKey)